_docs/_latest/develop/website.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Install, Build, and Run Forseti Website
+order: 200
+---
+
+#  {{ page.title }}
+
+The instructions below are aimed at developers using Linux, particularly Debian
+or Ubuntu. This pages shows you how to install and run the website server
+lcoally, so that you can test your documentation changes before merging.
+
+---
+
+## High-Level Overview
+
+To build the Forseti Security website locally:
+
+1. [Install Docker](#install-docker)
+2. [Install Ruby & RubyGems](#install-ruby)
+3. [Install Bundler](#install-bundler)
+4. [Install Jekyll](#install-jekyll)
+5. [Fetch, Build, and Run Forseti Web Code](#fetch-code)
+
+## Step by Step Instructions
+
+### Install Docker {#install-docker}
+
+```bash
+sudo apt-get update
+sudo apt-get install docker-ce
+```
+
+NOTE: If Docker is not available in the repository, follow
+the steps [here from the Docker
+website](https://docs.docker.com/install/linux/docker-ce/ubuntu/#set-up-the-repository)
+to add Docker to the package index.
+
+NOTE: If the previous command fails even after ensuring that Docker
+is in the package repository list, you may need to clear your
+docker lib (rm -rf /var/lib/docker) as well. Doing this will delete
+all images on disk. Ensure that they are backed up somewhere
+if you don't want this to happen.
+
+In order to avoid permission issues later on, add your user to the
+``docker`` group:
+
+```bash
+sudo adduser $USER docker
+```
+
+Before continuing, test that Docker is running correctly by
+doing the following:
+
+```bash
+docker run hello-world
+```
+
+### Install Ruby & RubyGems {#install-ruby}
+
+To install Ruby and RubyGems, execute the following:
+
+```bash
+sudo apt-get install ruby ruby-dev
+```
+
+### Install Bundler {#install-bundler}
+
+```bash
+gem install bundler
+```
+
+You may encounter the following error:
+
+```bash
+ERROR:  While executing gem ... (Gem::FilePermissionError)
+You don't have write permissions for the /var/lib/gems/2.3.0 directory.
+```
+
+If you do, the solution is [here](https://stackoverflow.com/a/47207118/1783829)
+and is listed below:
+
+```bash
+cd /var/lib
+sudo chmod -R a+w gems/
```

Comment:
I believe this can be fixed with a ruby virtual env, like rbenv. That will also make it easier for users to not have to install gems at system level, they localized to the user and virtual envs at that.